invenio_cli/helpers/versions.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 CERN.
+# Copyright (C) 2025 TU Wien.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio CLI dependencies helper."""
+
+import os
+import re
+
+import tomli
```

Comment:
`tomllib` is part of the standard lib starting with python 3.11, but until then we can use `tomli` which is a dependency from somewhere (`pipfile`?)

Reply:
Maybe we add a direct dependency to `tomllib` with a Python version specifier? On my machine I get just a `toml` dependency when e.g. running on Python 3.9:

```
toml v0.10.2
└── pipfile v0.0.2 [requires: toml *]
    └── invenio-cli v1.6.0 [requires: pipfile >=0.0.2]
```

Reply:
right, i actually forgot that you can add dependencies for specific python versions only

Reply:
leaving this here for now: https://github.com/hukkin/tomli?tab=readme-ov-file#building-a-tomlitomllib-compatibility-layer